src/main/groovy/org/pih/warehouse/outbound/ImportPackingListItem.groovy

```diff
@@ -121,7 +140,7 @@ class ImportPackingListItem implements Validateable {
         })
         binLocation(nullable: true, validator: { Location binLocation, ImportPackingListItem item ->
             if (!item.binLocationFound) {
-                return ['binLocationNotFound', binLocation.name]
+                return ['binLocationNotFound', binLocation?.name]
```

Comment:
I don't mind it, but I'm wondering if you had run into the nullpointer here? I think it's not possible, because the only possible option to execute this code is if you return the dummy location

Reply:
you are right, I had a NPE but I had used a different approach before where I returned a a `null` value with setting this flag as `false` which resulted in a NPE.
I can remove this nullsafe operator now